index.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>My Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <header>
+        <ul>
+            <li> <p class="welcome">Welcome</p></li>
+            <li> <img src="img/union.png" alt="menu bar"> </li>
+        </ul>
+            
+            
+
+    </header>
+
+
+    <section>
+        <p class="introduction"> Hey there. I’m Chukwuemeka <br> <span> I’m a software developer</span></p>
```

Comment:
- [ ] This is the main title of your page. For that reason, `h1` tag is the best tag to use here. The reason is If you want that google (and other search engines) knows about a specific topic, in this case, you, use an `h1` tag. [Check here](https://www.reliablesoft.net/h1-tag/) for more info